content/en/docs/ambient/architecture/data-plane/index.md

```diff
@@ -0,0 +1,104 @@
+a user can set a policy requiring an identity (either *any* identity, or a specific one) to block all plaintext traffic.
+
+When the destination is waypoint enabled, if the source is in ambient mesh, the source's ztunnel ensures the request **must** go through
+the waypoint where policy is enforced. However, a workload outside of the mesh doesn't know anything about waypoint proxies therefore it sends
+requests directly to the destination without going through any waypoint proxy even if the destination is waypoint enabled.
+Currently, traffic from sidecars and gateways won't go through any waypoint proxy either and they will be made aware of waypoint proxies
+in a future release.
+
+#### Dataplane details
+
+The L4 ambient dataplane between is depicted in the following figure.
+
+{{< image width="100%"
+link="ztunnel-datapath-1.png"
+caption="Basic ztunnel L4-only datapath"
+>}}
+
+The figure depicts ambient pod workloads running on two nodes W1 and W2 of a Kubernetes cluster. There is a single instance of the ztunnel proxy on each node. In this scenario, application client pods C1, C2 and C3 need to access a service provided by pod S1 and there is no requirement for advanced L7 features such as L7 traffic routing or L7 traffic management so no Waypoint proxy is needed.
+
+The figure shows that pods C1 and C2 running on node W1 connect with pod S1 running on node W2 and their TCP traffic is tunneled through {{< gloss >}}HBONE{{< /gloss >}} tunnel instances that have been created by the ztunnel proxy pods of each node. Mutual TLS (mTLS) is used for encryption as well as mutual authentication of traffic being tunneled. SPIFFE identities are used to identify the workloads on each side of the connection. For more details on the tunneling protocol and traffic redirection mechanism, refer to the guides on [HBONE](/docs/ambient/architecture/hbone) and [ztunnel traffic redirection](/docs/ambient/architecture/traffic-redirection).
+
+{{< tip >}}
+Note: Although the figure shows the HBONE tunnels to be between the two ztunnel proxies, the tunnels are in fact between the source and destination pods. Traffic is HBONE encapsulated and encrypted in the network namespace of the source pod itself, and eventually decapsulated and decrypted in the network namespace of the destination pod on the destination worker node. The ztunnel proxy still logically handles both the control plane and data plane needed for HBONE transport; however, it is able to do that from inside the network namespaces of the source and destination pods.
+{{< /tip >}}
+
+Note that the figure shows that local traffic - from pod C3 to destination pod S1 on worker node W2 - also traverses the local ztunnel proxy instance so that L4 traffic management functions such as L4 Authorization and L4 Telemetry are enforced identically on traffic, whether or not it crosses a node boundary.
+
+## In Mesh routing with Waypoint enabled
+
+A waypoint exclusively receives HBONE requests.
```

Comment:
ztunnel will only send hbone to the ref impl waypoints at least but they can handle other traffic. Maybe we can just remove the sentence though rather than trying to clarify it...
```suggestion

```

Reply:
Yeah good point.

Reply:
I do think it's important to reinforce that this comms channel is secured tho, so I might go with

`Istio waypoints exclusively receive HBONE traffic`